### PR TITLE
add K = 'jump to link' in visual mode

### PR DESCRIPTION
**Pull request by Konfekt**
Solves https://github.com/lambdalisue/vim-manpager/issues/6

### CODE REVIEW

File: ftplugin/man.vim [hunk 1]
```diff
@@ -12,11 +12,13 @@ setlocal iskeyword+=\.,-
 nnoremap <buffer><silent> <Plug>(manpager-next-keyword)     :<C-u>call manpager#find_next_keyword()<CR>
 nnoremap <buffer><silent> <Plug>(manpager-previous-keyword) :<C-u>call manpager#find_previous_keyword()<CR>
 nnoremap <buffer><silent> <Plug>(manpager-open)             :<C-u>call manpager#open('', expand('<cword>'))<CR>
+xnoremap <buffer><silent> <Plug>(manpager-open)             :<C-u>call manpager#open('', '<C-R>=manpager#get_visual_selection()<CR>')<CR>
```
Comment (lambdalisue) [inline ftplugin/man.vim:15]: Why don you just call `manpager#get_visual_selection()` directly?

**Comment by Konfekt** [comment]
I'm not sure I understand?! 

If you mean an `<expr>` mapping, then it is because the `get_visual_selection()` only works with `<c-r>=`.

**Comment by lambdalisue** [comment]
You right.

**Comment by lambdalisue** [comment]
Thanks a lot :-)

**Pull request merged**